### DISCUSSION TITLE
It doesn't work in desktop at all

### DISCUSSION THREAD

**Discussion by regs01** [mozilla/firefox-translations#719] [General]
Just toolbar pops up and nothing else happening. Whatever language you choose. Firefox 113.

**Comment by regs01** [comment]
Oh there is a translate button which is not visible in light theme.

**Comment by marco-c** [comment]
We are not going to make any improvements or fixes to the addon since we are now focusing on the built-in version.
You can report problems with the built-in version on Bugzilla (https://bugzilla.mozilla.org/).
